### PR TITLE
NDS-138 Callout component

### PR DESCRIPTION
**Pull request by wa-rren-dev**
https://nicedigital.atlassian.net/browse/NDS-138

### CODE REVIEW

File: components/nds-callout/scss/callout.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
+////
+/// @group components
+////
+
+@import '~@nice-digital/nds-core/scss/core';
+
+.callout {
+  background: $nds-colour-nice-white;
+  height: auto;
+  margin-bottom: rem($nds-spacing-large);
```
Comment (ediblecode) [inline components/nds-callout/scss/callout.scss:10]: Shouldn't this have `position: relative` so the anchor `:before` fills the space?

File: components/nds-callout/scss/callout.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  }
+  &__body {
+    padding: 1rem;
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6 {
+      margin-top: 0;
```
Comment (ediblecode) [inline components/nds-callout/scss/callout.scss:22]: why don't we do a `> :first:child` with no margin top and a `> :last:child` with no margin-bottom like we do with panel, and then we don't need the list of headings

File: components/nds-callout/scss/callout.scss [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+.callout {
+  background: $nds-colour-nice-white;
+  height: auto;
+  margin-bottom: rem($nds-spacing-large);
+  &__image img {
+    width: 100%;
+  }
+  &__body {
+    padding: 1rem;
```
Comment (ediblecode) [inline components/nds-callout/scss/callout.scss:15]: Should this be `rem($nds-spacing-medium)`?

File: components/nds-callout/scss/callout.scss [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6 {
+      margin-top: 0;
+    }
+    a {
+      text-decoration: none;
```
Comment (ediblecode) [inline components/nds-callout/scss/callout.scss:25]: Should we have an underline when you hover?

File: components/nds-callout/src/Callout.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+		<div className={classnames("callout__body", className)} {...rest}>
+			{children}
+		</div>
+	);
+};
+
+export const CalloutImage = ({ className, children, ...rest }) => {
+	return (
+		<div className={classnames("callout__image", className)} {...rest}>
+			<MaintainRatio ratio="16:9">{children}</MaintainRatio>
```
Comment (ediblecode) [inline components/nds-callout/src/Callout.js:26]: Oooh I like the use of the other component

File: components/nds-callout/src/Callout.js [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+		</div>
+	);
+};
+
+Callout.propTypes = {
+	children: PropTypes.node,
```
Comment (ediblecode) [inline components/nds-callout/src/Callout.js:32]: we could add isrequired on the prop types for the 3 children props I think

Comment (ediblecode) [reply components/nds-callout/src/Callout.js:32]: This would match up with the type defs

**Review by ediblecode — CHANGES_REQUESTED** [review]

File: components/nds-callout/scss/callout.scss (hunk 1 above)
Comment (wa-rren-dev) [reply components/nds-callout/scss/callout.scss:10]: So without relative it just expands to the grid so I didn't notice this!

**Review by ediblecode — APPROVED** [review]

**Pull request merged**